docs/advanced/addons/rancher-vcluster.md

```diff
@@ -10,7 +10,7 @@ title: "Rancher vcluster (Experimental)"
 
 _Available as of v1.2.0_
 
-The `rancher-vcluster` addon allows you to run Rancher as a workload on the underlying Harvester cluster using [vcluster](https://www.vcluster.com/).
+The `rancher-vcluster` addon allows you to run Rancher Manager as a workload on the underlying Harvester cluster and is implemented using [vcluster](https://www.vcluster.com/).
```

Comment:
```suggestion
The `rancher-vcluster` add-on allows you to run Rancher Manager as a workload on the underlying Harvester cluster and is implemented using [vcluster](https://www.vcluster.com/).
```

Reply:
we will stick with `addon` for now since UI is already showing `Addon`